Linux/sgx/test_app/enclave/tests/threadstest.c

```diff
@@ -56,32 +56,71 @@
 
 #define fprintf(stream, msg...) printf(msg)
 
-typedef pthread_t thread_t;
-
 sgx_status_t ucreate_thread();
 
-/*
-static void *thread_run(void *arg)
+void (*func)(void) = NULL;
+volatile int busy_wait = 0;
+
+#if !defined(OPENSSL_THREADS) || defined(CRYPTO_TDEBUG)
+
+typedef unsigned int thread_t;
+
+static int run_thread(thread_t *t, void (*f)(void))
+{
+       int res = 0;
+       func = f;
+       busy_wait = 1;
+       sgx_status_t status = ucreate_thread(&res);
+       if (status != SGX_SUCCESS || res != 0)
+               return 0;
+       return 1;
+}
+
+static int wait_for_thread(thread_t thread)
+{
+	while (busy_wait == 1);
+	return 1;
+}
+
+#elif defined(OPENSSL_SYS_WINDOWS)
```

Comment:
The test code was ported from OpenSSL. I thought it would be OK to use a same file for both OSes. But it looks thread test was not enabled at Windows part. I can remove these lines. Thanks.